src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.RemoteHostClientService.cs

```diff
@@ -213,26 +207,6 @@ private void OnConnectionChanged(object sender, bool connected)
                 FatalError.Report(new Exception("Connection to remote host closed"));
             }
 
-            private bool FeaturesEnabled()
-            {
-                if (ServiceFeatureOnOffOptions.IsClosedFileDiagnosticsEnabled(_workspace.Options, LanguageNames.CSharp))
-                {
-                    return true;
-                }
-
-                if (DynamicAnalysisEnabled())
-                {
-                    return true;
-                }
-
-                if (CodeLenEnabled())
```

Comment:
If someone turned on CodeLens later, we would not properly support running it out of proc.